lib/pages/list/task_edit.dart

```diff
@@ -221,15 +220,15 @@ class _TaskEditPageState extends State<TaskEditPage> {
                         ),
                         onTap: () {
                           // We add a new entry every time we add a new input, to make sure all inputs have a place where they can put their value.
-                          _reminderDates!.add(null);
-                          var currentIndex = _reminderDates!.length - 1;
+                          _reminderDates.add(DateTime(0));
```

Comment:
Adding a `null` threw an exception even if the list items were nullable.

Reply:
Weird. It worked for me. But this is cleaner